README.md

```diff
@@ -162,6 +164,16 @@ key: "ENABLE_NVFUSER"
 }
 ```
 
+* Additional Optimizations: Three additional boolean parameters are available to disable
+certain Torch optimizations that can sometimes cause latency regressions in models with
+complex execution modes and dynamic shapes. If not specified, all are enabled by default.
+
+    `ENABLE_JIT_EXECUTOR`
```

Comment:
Maybe the user would benefit from a short description of what the significance of each flag is. Is there pytorch docs we can point to for these?

Reply:
I looked some but didn't find anything...

Reply:
I couldn't find anything either. Internal experts pointed me to the Python versions of these APIs to try disabling to deal with our issue.